src/vue/components/panel.vue

```diff
@@ -60,20 +60,33 @@
         </div>
         <div v-if="$store.getters['app/defDataReady']">
           <div :key="definition.ID"
-               class="alpheios-panel__contentitem" v-for="definition in formattedShortDefinitions">
-            <shortdef :definition="definition" :languageCode="$store.state.app.languageCode"></shortdef>
+               class="alpheios-panel__contentitem"
+               v-for="definition in formattedShortDefinitions"
```

Comment:
May be it is better add v-key here?

Reply:
I totally agree that `v-key`s are better. The strange thing is that I see a `v-key` in the code (https://github.com/alpheios-project/components/blob/comp-i365/src/vue/components/panel.vue#L62-L65) but it is not shown here. That's definitely weird.

Reply:
oh may be it is in a hidden row, than it is ok!